docs/sources/explore/logs-integration.md

```diff
@@ -97,6 +97,16 @@ You can change the order of received logs from the default descending order (new
 
 Each log row has an extendable area with its labels and detected fields, for more robust interaction. For all labels we have added the ability to filter for (positive filter) and filter out (negative filter) selected labels. Each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.
 
+### Escaping newlines
+
+> **Note:** Available in Grafana 7.5 and later versions.
+
+Explore can automatically detect and attempt to fix incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). This feature appears only when Explore detects incorrectly escaped sequences.
```

Comment:
```suggestion
Explore can automatically detect and attempt to fix incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). 
```

Reply:
This feature won't be visible in many circumstances. I'm concerned that omitting the conditional nature (and concerns about reliability that led to calling it "experimental") could confuse users who expect it to be visible. Is there a better way to emphasize this?

Reply:
How about "The "Escape newlines" option is available only when Explore detects incorrectly escaped sequences."

Reply:
Also, this capability is probably no longer "experimental". Might be worth it to find out more.

Reply:
> Also, this capability is probably no longer "experimental". Might be worth it to find out more.

The linked issue has some discussion about this. "Experimental" here referred to the relative reliability of detection and replacement, not the supportability of the feature.

> I would actually remove the note "Available in Grafana 7.5 and later versions".

👍🏻